lib/__tests__/help.test.js

```diff
@@ -0,0 +1,115 @@
+const chalk = require('chalk');
+
+const { helpCommand, findCommandByName } = require('../help');
+/* eslint-disable no-console */
+console.log = jest.fn();
+console.error = jest.fn();
+/* eslint-enable no-console */
+
+describe('helpCommand', () => {
+  beforeEach(() => {
+    /* eslint-disable no-console */
+    console.log.mockClear();
+    console.error.mockClear();
+    /* eslint-enable no-console */
+  });
+
+  const mockCommands = {
```

Comment:
Just curious about the decision to use test data here instead of grabbing commands from `paragon-scripts`.

I can see pros and cons to both approaches. If we decide it makes sense to continue to use test data here, it'd be good to cover a few more of the use cases that the commands in `paragon-scripts` cover:
* `options` with `choices`
* Multi-line descriptions (`--themesPath` is a big one)

Reply:
I decided to import commands from `paragon-scripts` and also added the necessary tests